contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -51,32 +52,44 @@ const (
 	Deleted  = FlagType("deleted")
 )
 
+var defaultRoles = []string{"*"}
+
 // A struct that describes a pod task.
 type T struct {
-	ID          string
-	Pod         api.Pod
-	Spec        Spec
+	ID  string
+	Pod api.Pod
+
+	// stores the final procurement result, once set read-only
```

Comment:
nit: can we have punctuation at the end of this line so that the HTML docs are more readable?